src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.Android/Airship.cs

```diff
@@ -377,5 +400,9 @@ public bool OnDeepLink(string deepLink)
             handler(deepLink);
             return true;
         }
+
+        public void OnInboxUpdated()
+        {
```

Comment:
Need to add ```
//Adding Inbox updated listener
            EventHandler handler = OnMessageCenterUpdated;
            if (handler != null)
            {
                handler(this, EventArgs.Empty);
            }
```
here since this is the Airship callback when the inbox is updated.